crates/ruff_server/src/session/index.rs

```diff
@@ -200,16 +200,20 @@ impl Index {
         workspace_settings: Option<ClientSettings>,
         global_settings: &ClientSettings,
     ) -> crate::Result<()> {
+        if workspace_url.scheme() != "file" {
+            tracing::info!("Ignoring non-file workspace URL: {workspace_url}");
```

Comment:
Logging an info message is probably fine, but I could see users being confused by ruff isn't working in a workspace. What do you think of showing a warning message?

Reply:
Yeah, that seems reasonable.